proto/v2/resolve.proto

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+syntax = "proto3";
+package datacommons.v2;
+
+option go_package = "github.com/datacommonsorg/mixer/internal/proto/v2";
+
+// Three modes are supported:
+//
+// 1. ID to ID:
+//   nodes = ["wikidataId/CA", "wikidataId/MA"]
+//   property = "<-wikidataId->nutsCode"
+//
+// 2. Coordinate (latitude#longitude) to DCID:
+//   nodes = ["34.56#-23.12", "-22.11#44.55"] 
+//   property = "<-geoCoordinate"
```

Comment:
Here have an implicit dcid like "<-geoCoordinate->dcid". I wonder should make that explicit, so it matches 1) where other ids are used?

Then the response is also clear that resolved_ids are the ids asked in the request.

Reply:
SG - done.